packages/suspense-website/package.json

```diff
@@ -26,8 +26,8 @@
     "lodash.shuffle": "^4.2.0",
     "match-sorter": "latest",
     "playwright": "latest",
-    "react": "latest",
-    "react-dom": "latest",
+    "react": "experimental",
+    "react-dom": "experimental",
```

Comment:
This feels unfortunate, but maybe there's a trade off in clarity– since the library (and the feature it's building on top of) are also _experimental_.